highfidelity/hifi
# Daily tests

# Description
1.  Adds a method to the Test class that maximizes the window.  This is needed so that snapshots taken with the primary camera will have a constant size.
2.  Added options to AutoTester. to show and hide the Windows task bar.
# Testing
1.  Make sure the local sandbox is running
2.  Run interface from the command line with any test script
     `interface.exe --testScript D:\GitHub\hifi\scripts\defaultScripts.js`
3.  Resize the Interface window to some size (e.g. half the screen size).
4.  Open the Scripting Console (CTRL+ALT+J)
5.  Enter `Test.showMaximized();` and verify that the window becomes full screen
